src/main/kotlin/com/autonomousapps/DependencyAnalysisExtension.kt

```diff
@@ -57,6 +58,11 @@ open class DependencyAnalysisExtension @Inject constructor(project: Project) : A
     action.execute(issueHandler)
   }
 
+  /** Customize project properties. See [ProjectHandler] for more information. */
+  fun projectProperties(action: Action<ProjectHandler>) {
+    action.execute(projectHandler)
+  }
```

Comment:
Not sure about this method name or using this as the access point.

Reply:
I would be ok with exposing this new property in the top-level DSL object (`dependencyAnalysis`).

If you file (or discover) an issue for auto-detecting this, please link it to this PR.

Reply:
Gradle has confirmed to me directly that there is no API to detect this. Nevertheless I think it would be a better user experience if it were automatic. How do you feel about parsing the settings script with regex, looking for that specific line that enables the feature?

I also wonder if we can inspect the buildscript classloader for a specific jar or class file, where these accessors would live?

Reply:
Sounds good. Yeah, my worry was that Gradle would remove the need for the `enableFeaturePreview` syntax once typesafe project accessors were promoted to stable.

I'll go ahead and move the property to top level.